registry_no_code_module.go

```diff
@@ -246,3 +301,14 @@ func (o *RegistryNoCodeModuleUpdateOptions) valid() error {
 func (o *RegistryNoCodeModuleReadOptions) valid() error {
 	return nil
 }
+
+func (o *RegistryNoCodeModuleCreateWorkspaceOptions) valid() error {
+	if !validString(o.Name) {
+		return ErrRequiredName
+	}
+	if !validStringID(o.Name) {
+		return ErrInvalidName
+	}
```

Comment:
validStringID ensures the field follows the resource external ID format used by our API (e.g for a workspace it would be `ws-1234abcde`).

So either this field accepts a workspace's external ID or a org/workspace name pair to identify a workspace, but not both. 

However since this creates a workspace using a no-code module, my guess would be to provide the name not an external ID? In which case we only need to call `validString(o.Name)`

Reply:
@sebasslash thanks for the feedback! I've addressed this and your other comment in [this commit](https://github.com/hashicorp/go-tfe/pull/927/commits/e8f7b3144ee48aabe3a327ed2c59e3972da73403).